### PR TITLE
make optimization parameters default

### PR DESCRIPTION
**Pull request by tst-basilhuffman**
initial commit

### CODE REVIEW

File: earth_enterprise/src/fusion/autoingest/plugins.src/PacketLevel.src [hunk 1]
```diff
@@ -81,14 +81,7 @@ void PacketLevelAssetVersionImplD::DoSubmitTask(void) {
...
 
-  // misc.xml now contains an option, DisablePacketLevelVersionCachePurge, which
-  // allows the user to disable cache purging. The default value is false
-
-  if (! MiscConfig::Instance().DisablePacketLevelVersionCachePurge)
-  {
-     MutableAssetVersionD::PurgeCacheIfNeeded(
-       (taskConfig.insets.size() << 1) + 32);
-  }
+  MutableAssetVersionD::PurgeCacheIfNeeded((taskConfig.insets.size() << 1) + 32);
```
[the remark this answers is not in the document]
Comment (tst-lsavoie) [reply earth_enterprise/src/fusion/autoingest/plugins.src/PacketLevel.src]: We may also be able to completely remove the PurgeCacheIfNeeded function if it isn't called from anywhere else.

**Comment by tst-lsavoie** [comment]
Looks like this may need to be rebased - there are a lot of commits showing up on this PR.

**Comment by tst-basilhuffman** [comment]
> Looks like this may need to be rebased - there are a lot of commits showing up on this PR.

everything looked ok when rebasing but those commits are still there, so im not sure how to get rid of them

File: docs/geedocs/5.3.0/answer/176738.html [hunk 2]
```diff
@@ -255,10 +255,10 @@ <h2 id="cache_configuration">Cache configuration</h2>
 (assuming your asset root is <code>/gevol/assets</code>; if not, modify the path accordingly).
 The asset cache size is controlled by <code>AssetCacheSize</code> and the asset version cache size is controlled by <code>VersionCacheSize</code>.
 Note that the asset version cache is generally larger than the asset cache because it may contain multiple versions of the same asset.</p>
-
-Also included within <code>misc.xml</code> is an experimental option, which allows for the user to disable cache purging while projects build. This is controlled by <code>DisablePacketLevelVersionCachePurge</code> and its default value is <code>false</code>.
 <br><br>
-In some instances, memory utilization may become problematic. An experimental feature is included which will allow for modifying aspects of the the XML cache. This is controlled through <code>/etc/opt/google/XMLparams</code>. It takes the form of a standard configuration file, and it is in the following format.
+Within <code>misc.xml</code>, the following options are deprecated, as enabled permanently: <code>DisablePacketLevelVersionCachePurge</code> and <code>ConsolidateListenerNotifications</code>.
```
Comment (tst-lsavoie) [inline docs/geedocs/5.3.0/answer/176738.html]: Minor: might change "as enabled permanently" to "as if they were enabled permanently".

**Pull request merged**